generics/middleware/authenticator.js

```diff
@@ -99,7 +99,7 @@ module.exports = async function (req, res, next) {
     return
   }
 
-  let internalAccessApiPaths = ["/cloud-services/","apps/create","apps/update"];
+  let internalAccessApiPaths = ["/cloud-services/","apps/create","apps/update","list"];
```

Comment:
@aman-tunerlabs - List is common, you should which module list here

Reply:
@priyanka-TL  PLease add only the lists apis which is been used by unnati .

Reply:
@aman-tunerlabs resolved